src/components/submission/list.vue

```diff
@@ -219,6 +219,12 @@ export default {
       }
       return conditions.length !== 0 ? conditions.join(' and ') : null;
     },
+    odataSelect() {
+      if (this.selectedFields == null) return null;
+      const paths = this.selectedFields.map(({ path }) => path.replace('/', ''));
```

Comment:
The fields' paths are already mostly valid OData paths because Frontend requests &hellip;/fields?odata=true. However, we still need to remove the leading slash.